kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -2170,4 +2172,19 @@ public String getCamsComplexMaintenanceDocumentLookup() {
         return managedLists;
     }
 
+    public List<Note> getBoNotes() {
+        if (!StringUtils.isEmpty(getObjectId())) {
+            boNotes = SpringContext.getBean(NoteService.class).getByRemoteObjectId(getObjectId());

```

Comment:
While the NoteService class is generated by Spring, AssetBase is not, because the Assets are being generated by ojb.

I will modify the code to fit the pattern given in the Coding standards